src/classes/CON_EthnicityRace_TEST.cls

```diff
@@ -0,0 +1,80 @@
+/*
+    Copyright (c) 2017, Salesforce.org
+    All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2017
+* @group Contacts
+* @group-content ../../ApexDocContent/Contacts.htm
+* @description Tests Race Ethnicity Batch Class
+*/
+@isTest
+private class CON_EthnicityRace_TEST {
+
+	Private static List<String> races = new List<String>{
+			'Non-Standard to see what happens',
+			'White',
+			'Asian',
+			'Hispanic or Latino',
+			'Not Hispanic or Latino',
+			'Black or African American',
+			'American Indian or Alaskan Native',
+			'Native Hawaiian or Other Pacific Islander'};
+
+	@testSetup
+	static void dataSetup() {
+
+		// How many races to test
+		Integer raceCount = races.size();
+
+		// Get contacts
+        List<Contact> listCon = UTIL_UnitTestData_TEST.getMultipleTestContacts(raceCount);
+        for (Integer i = 0; i < raceCount; i++) {
+            listCon[i].LastName = 'ToAvoidDupeRule' + i;//Avoid duplicate matching
+
+            // Setup the Ethnicity info
+            listCon[i].Ethnicity__c = races.get(i);
+        }
+
+        insert listCon;
+	}
+
+	@isTest
+	static void testEthnicityRaceBatch() {
```

Comment:
The dataSetup method uses the @testSetup annotation, which creates data and stores it for all tests.